src/bench/base58.cpp

```diff
@@ -13,7 +13,7 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
```

Comment:
Yeah but is that the only reason clang is giving that warning? I agree that it would be pedantic to add an extra element to the array just to get rid of that warning, seems very hacky